CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 This file is used to list changes made in each version of vcruntime.
 
+## 0.2.14:
```

Comment:
Can we squash this all down to a single version. I think it's safe to call this 1.0 at this point.